src/cpp/handler/handlerapp.cpp

```diff
@@ -238,6 +262,7 @@ class HandlerApp::Private : public QObject
 		trimlist(&condure_in_stream_specs);
 		QStringList condure_out_specs = settings.value("proxy/condure_out_specs").toStringList();
 		trimlist(&condure_out_specs);
+		int proxyWorkerCount = settings.value("proxy/workers", 1).toInt();
```

Comment:
My one concern is hardcoding this value here 
It would be neat to fetch from a global var instead

Reply:
you mean like make some `#define DEFAULT_PROXY_WORKER_COUNT 1` ?

Reply:
ok will consider a follow up that does this for all such default values